packages/backend/src/core/remote/activitypub/ApResolverService.ts

```diff
@@ -116,6 +117,10 @@ export class Resolver {
 			throw new Error('cannot resolve already resolved one');
 		}
 
+		if (this.history.size > this.recursionLimit) {
+			throw new Error('hit recursion limit');
```

Comment:
```suggestion
			throw new Error(`hit recursion limit: ${this.utilityService.extractDbHost(value)}`);
```

I think appending the hostname may help for moderation.